201-front-door-premium-app-service-private-link/modules/front-door.bicep

```diff
@@ -56,7 +56,8 @@ resource profile 'Microsoft.Cdn/profiles@2020-09-01' = {
 }
 
 resource endpoint 'Microsoft.Cdn/profiles/afdEndpoints@2020-09-01' = {

```

Comment:
Just noticed this....  if I have the parent property, do I still need the full resourceType?

Reply:
Yes - if you're using the parent property then you do need a fully qualified resourceType. If you use a nested resource then you don't - but stylistically I generally prefer to avoid nested resources (with a few exceptions) since they make the template harder to read.

Reply:
ack - I don't like nesting either